toolchain/sem_ir/inst_kind.h

```diff
@@ -80,6 +80,10 @@ enum class InstConstantKind : int8_t {
   // the constants block.
   // TODO: Decide if this is the model we want for these cases.
   Unique,
+  // Same as unique, except that the instruction may or may not be constant.
```

Comment:
```suggestion
  // Similar to `Unique`, except that the instruction may or may not be constant.
```

It's different so not the same. ;)

Reply:
For me, "Same as X except Y" is saying the only difference is Y, whereas "Similar to X except Y" leaves open the possibility that there are other differences too (it's mostly similar to X, but in case Y it's not even similar).

How about:
```suggestion
  // The instruction may be a unique constant, as described for `Unique`.
  // Otherwise the instruction is not constant.
```

Reply:
FWIW, mostly I wanted "unique' backticked, the "same" doesn't bother me that much.

Reply:
FWIW, maybe also `Unique` -> `AlwaysUnique`?

Reply:
Renamed.